group_vars/approvals/staging.yml

```diff
@@ -29,3 +29,61 @@ pg_hba_postgresql_user: "all"
 pg_hba_postgresql_database: "all"
 pg_hba_method: "md5"
 pg_hba_source: "{{ ansible_host }}/32"
+
+# signoz
+otel_default_resource_attributes:
+  host.name: "{{ inventory_hostname }}"
+  service.name: approvals
+  service.version: staging
+  environment: staging
+
+otel_receivers:
+  filelog/nginx-access:
+    include: ["/var/log/nginx/access.log*"]
```

Comment:
Will this end up with us sending multiple copies of the same log messages? IIRC the current log is just called `access.log` and the rotated logs are called `access.log<datestamp>`. If that's the case, then we would have sent the contents of yesterday's `access.log` yesterday, and we don't want to send the same content again today in a file with the datestamp in the name.